### PR TITLE
feat(web-vitals): Visually indicate failed measurement marks

### PR DESCRIPTION
**Pull request by dashed**
If the measurement marks have any failing web vitals, then indicate the measurement mark to be failing a threshold.

Screen Shot 2020-11-10 at 3 22 49 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/spans/measurementsPanel.tsx [hunk 1]
```diff
@@ -45,6 +46,20 @@ class MeasurementsPanel extends React.PureComponent<Props> {
             return null;
           }
 
+          const names = Object.keys(marks);
+          const records = Object.values(WEB_VITAL_DETAILS).filter(vital =>
+            names.includes(vital.slug)
+          );
+
+          const failedThreshold =
+            records.find(record => {
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/measurementsPanel.tsx:55]: Would using `some()` here be better?

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx [hunk 2]
```diff
@@ -370,6 +373,19 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
...
+          );
+
+          const failedThreshold =
+            records.find(record => {
+              const value = marks[record.slug];
+              if (typeof value === 'number') {
+                return value >= record.failureThreshold;
+              }
+              return false;
+            }) !== undefined;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx:387]: Should we have a a helper function for this logic? It appears twice in these changes.

File: src/sentry/static/sentry/app/components/events/interfaces/spans/measurementsPanel.tsx (hunk 1 above)
Comment (dashed) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/measurementsPanel.tsx:55]: That's definitely better.

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx (hunk 2 above)
Comment (dashed) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx]: I lifted this up. https://github.com/getsentry/sentry/pull/21910/commits/96de96da95607536130a89f70498a32def61ec62

**Review by doralchan — APPROVED** [review]

**Review by markstory — APPROVED** [review]

**Pull request merged**